docs/data/base/components/popper/PlacementPopper.js

```diff
@@ -107,7 +107,9 @@ export default function PlacementPopper() {
       <PlacementForm setPlacement={setPlacement} />
       <div style={{ padding: '4rem 0', textAlign: 'center' }}>
         <span
-          ref={(elm) => setAnchorEl(elm)}
```

Comment:
Can just do:

```
ref={setAnchorEl}
```

Reply:
In fact, I'm not sure this actually presents a problem since the setter returns `void`?

Reply:
Sold, we do this in other places, simpler.